libbeat/docs/security/users.asciidoc

```diff
@@ -185,71 +185,68 @@ endif::serverless[]
 [[privileges-to-publish-events]]
 ==== Grant privileges and roles needed for publishing
 
-Users who publish events to {es} need to create and read from {beatname_uc}
+Users who publish events to {es} need to create and write to {beatname_uc}
 indices. To minimize the privileges required by the writer role, you can use the
-<<privileges-to-setup-beats,setup role>> to pre-load dependencies. Then turn off
-setup options in the  {beatname_uc} config file before running {beatname_uc} to
-publish events. For example:
+<<privileges-to-setup-beats,setup role>> to pre-load dependencies.
 
 ifndef::no_ilm[]
-[source,yaml]
-----
-setup.template.enabled: false
-setup.ilm.check_exists: false
-setup.ilm.overwrite: false <1>
-----
-<1> Omit `ilm.check_exists` and `ilm.overwrite` if ILM is disabled.
-endif::no_ilm[]
+When using ILM, turn off the ILM setup check in the {beatname_uc} config file before
+running {beatname_uc} to publish events:
 
-ifdef::no_ilm[]
 [source,yaml]
 ----
-setup.template.enabled: false
+setup.ilm.check_exists: false
 ----
 endif::no_ilm[]
 
 To grant the required privileges:
 
-. Create a *writer role*, called something like +{beat_default_index_prefix}_writer+, that has
-the following privileges (this list assumes the setup options shown earlier are
-set to `false`):
+. Create a *writer role*, called something like +{beat_default_index_prefix}_writer+,
+that has the following privileges:
+
 +
+NOTE: Only the `monitor` cluster privilege and the `create` index privilege on
++{beat_default_index_prefix}-*+ indices are required in every configuration.
+
```

Comment:
The word "only" as a qualifier often confuses people. I would just say:

NOTE: The `monitor` cluster privilege and the `create` index privilege on
+{beat_default_index_prefix}-*+ indices are required in every configuration.

Reply:
What I wanted to express is not just that monitor+create are always required, but also that in the most minimal configuration *no other* privileges are required. I'll think a bit more about how to express that if possible.

Reply:
Understood. You might need to use more words to convey that. Let me know if you need help crafting something.

Reply:
Incidentally, I would not say `create` index privilege because users might misread that as `create_index`. Saying `create` on xyz indices is probably enough.